design-assets/cms-design-system-ui-kit.xml

```diff
@@ -9,9 +9,9 @@
     </image>
     <generator>Sketch</generator>
     <item>
-      <title>CMS Design System UI kit - v0.7</title>
-      <pubDate>Mon, 18 May 2020 15:11:00</pubDate>
-      <enclosure url="https://github.com/CMSgov/design-system/raw/master/design-assets/CMS-Design-System-UI-kit.sketch?raw=true" type="application/octet-stream" sparkle:version="0.7"/>
+      <title>CMS Design System UI kit - v2.0.0</title>
```

Comment:
@line47 does this look right for updating sketch? also is the library updated enough for v2?

Reply:
If there's updates to make we can hold off on updating sketch. I.e. what should we do with the focus styles in sketch?

Reply:
That looks right! Thanks, @bernardwang I think we are good to update Sketch